sdk/textanalytics/Azure.AI.TextAnalytics/samples/Sample_RecognizeHealthcareEntities.md

```diff
@@ -13,11 +13,9 @@ string apiKey = "<apiKey>";
 var client = new TextAnalyticsClient(new Uri(endpoint), new AzureKeyCredential(apiKey));
 ```
 
-## Recognizing healthcare entities in multiple documents
+## Get the Documents
 
-To recognize healthcare entities in multiple documents, call `StartAnalyzeHealthcareEntities` on an `IEnumerable` of strings.  The result is a Long Running operation of type `AnalyzeHealthcareEntitiesOperation` which polls for the results from the API.
-
-```C# Snippet:TextAnalyticsSampleHealthcareBatchConvenienceAsync
+```C#
```

Comment:
I find this somehow confusing because it is not clear where are we `getting` the documents from.
I think it is better if we merge this with the `Analyze Documents` section